test/mixins.py

```diff
@@ -13,7 +13,39 @@
 from pathlib import Path
 
 
-class TempCwdMixin(unittest.TestCase):
+class Mixin(unittest.TestCase):
```

Comment:
Do we need this class? Can't you just do `FileUtilsMixin(unittest.TestCase)`?

Reply:
Well, strictly it is not required. We even can drop inheritance from `unittest.TestCase` completely (but user/client need to remember to put `TestCase` as the last class in the inheritance list). New class `Mixin` is added as "self-documented" code (and also eventually some mro-check may be added here).